Update lab package doc paragraph (#4196)
jpmorganchase/salt-ds: site/docs/components/index.mdx

Stable components do not feature an accompanying symbol in their navigation link titles. You can find them in the `@salt-ds/core` package.

> The {meta.labsIcon} symbol denotes components that are under active development. These may not function as expected and may change due to feedback. You should exercise caution. You can find them in the `@salt-ds/lab` package. Please don't use lab components not listed on the website, which aren't receiving active development.
> 

Reviewer: maybe this? we use the wording in a few places, whatever sound less encouraging i guess 
```suggestion
The {meta.labsIcon} symbol denotes components that are under active development. These may not function as expected and may change due to feedback. You should exercise caution. You can find them in the `@salt-ds/lab` package. The use of lab components not listed on the website is not encouraged since they aren't receiving active development.

```

> The {meta.labsIcon} symbol denotes components that are under active development. These may not function as expected and may change due to feedback. You should exercise caution. You can find them in the `@salt-ds/lab` package. The use of lab components not listed on the website is not encouraged since they aren't receiving active development.

Reviewer 2: ```suggestion
The {meta.labsIcon} symbol denotes components that are under active development. These may not function as expected and may change due to feedback. You should exercise caution. You can find them in the `@salt-ds/lab` package.

**Note:** The use of lab components not listed on the website isn't encouraged since they aren't receiving active development.
```

thought's is not -> isn't is following our guidance
and should be make this stand out more?
